src/main/java/juja/sqlcmd/view/Console.java

```diff
@@ -3,7 +3,7 @@
 import java.util.Scanner;
 
 public class Console implements View {
-    Scanner scanner;
+    private Scanner scanner;
 
     public Console() {
```

Comment:
If you will delete a field then empty constructor will be enough. Also pay your attention that you initialize scanner field twice: once in a constructor and then in read() method